ldi-orchestrator/ldio-connectors/ldio-http-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioHttpIn.java

```diff
@@ -2,21 +2,31 @@
 
 import be.vlaanderen.informatievlaanderen.ldes.ldi.services.ComponentExecutor;
 import be.vlaanderen.informatievlaanderen.ldes.ldi.types.LdiAdapter;
+import be.vlaanderen.informatievlaanderen.ldes.ldi.types.LdiAdapter.Content;
 import be.vlaanderen.informatievlaanderen.ldes.ldi.types.LdiInput;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestHeader;
-import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.reactive.function.server.RouterFunction;
+import org.springframework.web.reactive.function.server.ServerResponse;
+
+import static org.springframework.web.reactive.function.server.RequestPredicates.POST;
+import static org.springframework.web.reactive.function.server.RouterFunctions.route;
 
-@RequestMapping
 public class LdioHttpIn extends LdiInput {
-	public LdioHttpIn(ComponentExecutor executor, LdiAdapter adapter) {
+	private final String endpoint;
+
+	public LdioHttpIn(ComponentExecutor executor, LdiAdapter adapter, String endpoint) {
 		super(executor, adapter);
+		this.endpoint = endpoint;
 	}
 
-	@PostMapping("data")
-	public void accept(@RequestBody String input, @RequestHeader("Content-Type") String contentType) {
-		getAdapter().apply(LdiAdapter.Content.of(input, contentType))
-				.forEach(getExecutor()::transformLinkedData);
+	public RouterFunction<ServerResponse> mapping() {
+		return route(POST("/%s".formatted(endpoint)),
+				req -> {
+					String contentType = req.headers().contentType().orElseThrow().toString();
+					return req.bodyToMono(String.class)
+							.doOnNext(content -> getAdapter().apply(Content.of(content, contentType))
+									.forEach(getExecutor()::transformLinkedData))
```

Comment:
Since the executor uses a separate thread to transform linked data, it will never be able to indicate whether the transformation is successful or not.
Would this not be a more suitable implementation: 
```
@Override
	public  synchronized void transformLinkedData(final Model linkedDataModel) {
			Model transformedLinkedDataModel = linkedDataModel;
			for (LdiTransformer component : ldiTransformers) {
				transformedLinkedDataModel = component.apply(transformedLinkedDataModel);
			}

			Model finalTransformedLinkedDataModel = transformedLinkedDataModel;
			ldiOutputs.parallelStream().forEach(ldiOutput -> ldiOutput.accept(finalTransformedLinkedDataModel));
	}
```